server/src/Api/DependencyInjection.cs

```diff
@@ -24,6 +24,11 @@ public static IServiceCollection AddApi(this IServiceCollection services)
             .AddControllers()
             .AddProblemDetailsConventions()
             .AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));
+        services
+            .AddProblemDetails(options => { options.MapToStatusCode<FailedValidationException>(StatusCodes.Status400BadRequest); })
```

Comment:
The added lines seem like a copy that shouldn't been here. Map the status code above, as it is already done. Also, validation error is 422